### ISSUE TITLE
Row borders missing in IE 11

### ISSUE THREAD

**Issue by iacii** [ag-grid/ag-grid#4452]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Providing a Reproducible Scenario**

An example can be found on the homepage: www.ag-grid.com

**Current behavior**

No borders appear between rows.

**Expected behavior**

Visible borders between rows.

**Please tell us about your environment:**

* **AG Grid version:** X.X.X

25.1.0. This wasn't a problem in version 21.

* **Browser:**

IE 11. It works correctly in Chrome.

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by icedjello** [comment]
Hi, 

Thank you for raising this. 
After looking into this this by comparing the page you linked in IE11 & Chrome, I was unable to see the difference you were referring to:
![image](https://user-images.githubusercontent.com/11807157/115544034-00fa5080-a29a-11eb-9560-b7d66350a594.png)

**Comment by icedjello** [comment]
Hi, 

Apologies for the mixup, this ticket was never meant to be close. 

I can confirm that we can see the issue you are talking about: 
https://ag-grid.atlassian.net/jira/software/projects/T2T/boards/14?selectedIssue=T2T-4471
We are looking into this and will follow up here with any updates.

Thanks!

**Comment by kiril-matev** [comment]
Hello!

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

This bug will be fixed as part of our next release - v25.2.

You can see the next release date on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

Thanks again for bringing this up with us - we appreciate your time!

**Comment by iacii** [comment]
Great, thank you for getting this fixed in the next version! I will be on the lookout for it.
